### PR TITLE
Encoding converter to UTF-8

### PR DESCRIPTION
**Pull request by RodicaMihaelaVasilescu**

### CODE REVIEW

File: ClangPowerTools/ClangPowerTools/ClangPowerTools.vsct [hunk 1]
```diff
@@ -428,6 +432,19 @@
...
+    </GuidSymbol>
+  
+    <GuidSymbol value="{6a93cd48-ec21-4054-a6f3-38cdbfef0ee5}" name="guidImages1">
+      <IDSymbol name="bmpPic1" value="1" />
+      <IDSymbol name="bmpPic2" value="2" />
+      <IDSymbol name="bmpPicSearch" value="3" />
+      <IDSymbol name="bmpPicX" value="4" />
+      <IDSymbol name="bmpPicArrows" value="5" />
+      <IDSymbol name="bmpPicStrikethrough" value="6" />
+    </GuidSymbol>
```
Comment (Enache-Ionut) [inline ClangPowerTools/ClangPowerTools/ClangPowerTools.vsct]: I think you don't need this `GuidSymbol` (`guidImages1` ). It's not used in another part of your changes.

File: ClangPowerTools/ClangPowerTools/Commands/CommandController.cs [hunk 2]
```diff
@@ -308,6 +314,29 @@ public void OnAfterRunCommand(object sender, CloseDataStreamingEventArgs e)
       {
         DisplayFinishedMessage(false);
       }
+
+      HasEncodingErrorEvent.Invoke(this, new EventArgs());
+    }
+
+    public void OnEncodingErrorDetected(object sender, HasEncodingErrorEventArgs e)
```
Comment (Enache-Ionut) [inline ClangPowerTools/ClangPowerTools/Commands/CommandController.cs]: Think about SOLID and separation of concerns principle for this method.

File: ClangPowerTools/ClangPowerTools/ClangPowerTools.vsct (hunk 1 above)
Comment (RodicaMihaelaVasilescu) [reply ClangPowerTools/ClangPowerTools/ClangPowerTools.vsct]: Yes, you're right. I will remove that.

File: ClangPowerTools/ClangPowerTools/MVVM/Command/RelayCommand.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+{
+  public class RelayCommand : ICommand
+  {
+    public event EventHandler CanExecuteChanged
+    {
+      add { CommandManager.RequerySuggested += value; }
+      remove { CommandManager.RequerySuggested -= value; }
+    }
+
+    private readonly Action _action;
```
Comment (Enache-Ionut) [inline ClangPowerTools/ClangPowerTools/MVVM/Command/RelayCommand.cs]: Please replace the `_` with the new standard notation for class members. As far as I know the standard naming for `actions` is using upper case for the first letter.

Comment (RodicaMihaelaVasilescu) [reply ClangPowerTools/ClangPowerTools/MVVM/Command/RelayCommand.cs]: Ok, I'll remove the '_' from '_action'. That variable is private, so I'll keep the lower case for the first letter.

File: ClangPowerTools/ClangPowerTools/Commands/CommandController.cs (hunk 2 above)
Comment (RodicaMihaelaVasilescu) [reply ClangPowerTools/ClangPowerTools/Commands/CommandController.cs]: Ok, I moved the method used for selecting items in ItemsCollector

**Pull request closed without merging**